src/foam/nanos/dig/DIG.js

```diff
@@ -99,9 +99,70 @@ foam.CLASS({
       label: 'URL',
```

Comment:
Does it make sense to still have this field if we are doing the same thing below? At the least, we shouldn't be duplicating the code.